metrics/core/lib/src/util/validation/validation_result.dart

```diff
@@ -0,0 +1,25 @@
+// Use of this source code is governed by the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+import 'dart:collection';
+
+import 'package:equatable/equatable.dart';
+import 'package:metrics_core/src/util/validation/target_validation_result.dart';
+import 'package:metrics_core/src/util/validation/validation_target.dart';
+
+/// A class that represents the result of [ValidationTarget]s' validation.
```

Comment:
🏷️ 
What do you think?
```suggestion
/// A class that represents the result of the whole validation process. It contains the validation results for each [ValidationTarget] participated in the process.
```